README.md

```diff
@@ -12,11 +12,14 @@ Aggregator service for insights results
 
 ## Description
 
-Insights Results Aggregator is a service that provides Insight OCP data that are being consumed by
-OpenShift Cluster Manager. That data contain information about clusters status (especially health,
-security, performance) based on results generated by Insights rules engine. Insights OCP data are
-consumed from selected broker, stored in a storage (that basically works as a cache) and exposed via
-REST API endpoints.
+Insights Results Aggregator is a service that provides Insight OCP data
+(offically called *recommendations*) that are being consumed by OpenShift
```

Comment:
typo:)

Reply:
But wasn't the new kid in the block "remediations"?

Reply:
fixed :)

Reply:
@joselsegura not now, probably next week perhaps :)